src/migrations/data.ts

```diff
@@ -12,16 +12,23 @@ import {
   WorkbookData,
 } from "../types/index";
 import { XlsxReader } from "../xlsx/xlsx_reader";
-import { migrationStepRegistry } from "./migration_steps";
+import { legacyMigrationStepRegistry } from "./migration_steps";
+import { migrationStepRegistry } from "./new_migration_steps";
 
-/**
- * This is the current state version number. It should be incremented each time
- * a breaking change is made in the way the state is handled, and an upgrade
- * function should be defined
- */
-export const CURRENT_VERSION = 25;
 const INITIAL_SHEET_ID = "Sheet1";
 
+export interface MigrationStep {
+  migrate: (data: any) => any;
+}
+
+export function getCurrentVersion() {
```

Comment:
technically, it's not the "current" version if there was no changes in one saas. Should it be rather named `getLatestVersion` ? or is it more confusing and `getCurrentVersion` is clearer even if it's not strictly speaking 100% correct in all cases?

Reply:
Should we add an empty migration step if there is no changes in one saas version to keep consistency ?

Reply:
In practice, I think we won't do it

Reply:
Except if we add it to the freeze checklist :D